v22.1/aws-dms.md

```diff
@@ -182,69 +155,60 @@ The `BatchApplyEnabled` setting can improve replication performance and is recom
 > SELECT table_catalog, table_schema, table_name, column_name FROM information_schema.columns WHERE is_hidden = 'YES';
 ~~~
```

Comment:
With the known limitations, are they on the main KL page? And do these need, or even have, GH issues? I don't know that it's necessary just pointing out.